environment.yml

```
@@ -3,58 +3,58 @@ channels:
 - defaults
 - conda-forge
 dependencies:
-- awscli=1.15.4
-- beautifulsoup4=4.5.3
-- botocore=1.10.4
-- boto=2.46.1
-- boto3=1.7.4
-- bqplot=0.10.2
-- cython=0.27.3
-- dask=0.17.2
+- awscli=1.16.96
+- beautifulsoup4=4.7.1
+- botocore=1.12.86
+- boto=2.49.0
+- boto3=1.9.86
+- bqplot=0.11.5
+- cython=0.29.4
```

Something in the list of conda packages has a `click` dependency; conda is installing `click=7.0.0`, which conflicts with civis-python's requirement of `click~6` (civisanalytics/civis-python#286). Let's explicitly install `click=6.7` here to avoid the conflict.
```suggestion
- click=6.7
- cython=0.29.4
```